codegen/types/api.py

```diff
@@ -119,10 +119,12 @@ def from_dict(cls, data: Dict[str, Any], docs_dir: Optional[str] = None) -> Api:
             filename = cls._gen_filename_from_classname(data["classname"])
 
         # Load the methods and the related docstrings if available
-        methods = [Method.from_dict(m) for m in data["methods"]]
```

Comment:
nit: if you want to avoid the `is not None` check I think you can do
```
methods = [Method.from_dict(m) for m in data.get("methods", [])]
```
I'm also not sure when/how you ran into methods being `None` here though